### ISSUE TITLE
Bug: --force-family not working for iPad only app

### ISSUE THREAD

**Issue by pachoo** [nowsecure/node-applesign#82]
The `--force-family` flag should be modifying the UIDeviceFamily for iPad only apps (UIDeviceFamily 2) to 1, however this isn't working in the case below.

Example Info.plist with `UIDeviceFamily` of 2.
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
  <dict>
    <key>CFBundleName</key>
    <string>TestApp</string>
    <key>CFBundleVersion</key>
    <string>1</string>
    <key>CFBundleInfoDictionaryVersion</key>
    <string>6.0</string>
    <key>CFBundleExecutable</key>
    <string>TestApp</string>
    <key>CFBundleIdentifier</key>
    <string>com.test.TestApp</string>
    <key>UIDeviceFamily</key>
    <array>
      <integer>2</integer>
    </array>
  </dict>
</plist>
```

After running applesign with the `--force-family` flag this is still 2.
```
$ applesign --force-family -i MY_CODESIGN_HASH -m MY.mobileprovision test.app  -o test_mod.ipa
// ...
UIDeviceFamily set to [2]
// ...
$ unzip -d x test_mod.ipa
$ cat x/Payload/TestApp.app/Info.plist
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
  <dict>
    <key>CFBundleName</key>
    <string>TestApp</string>
    <key>CFBundleVersion</key>
    <string>1</string>
    <key>CFBundleInfoDictionaryVersion</key>
    <string>6.0</string>
    <key>CFBundleExecutable</key>
    <string>TestApp</string>
    <key>CFBundleIdentifier</key>
    <string>com.test.TestApp</string>
    <key>UIDeviceFamily</key>
    <array>
      <integer>2</integer>
    </array>
  </dict>
</plist>
```

**Comment by pachoo** [comment]
This was with applesign 3.3.1 (as installed by npm).
```
$ applesign --version
3.3.1
```

**Comment by trufae** [comment]
should  be fixed  with 3.4,  please confirm

**Comment by trufae** [comment]
Closing for now, this issue was addressed already
